### PR TITLE
KED-2944: integrate the real API with experiment tracking frontend

### PR DESCRIPTION
**Pull request by studioswong**
## Description

This PR is the work to test and integrate the experiment tracking features with real data from the graphql endpoint.

## Development notes

<This is the work mainly to switch the reference of our graphql endpoint to the actual graphql endpoint on port 4141, as well as further minor changes to the queries to reflect the expected variable format for the endpoint. 

## QA notes

This will need to be integrated / tested with a kedro project that has a set of experiment runs generated. 

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

**Comment by studioswong** [comment]
The test is currently failing due to the python tests that are also failing on the main branch

File: src/components/experiment-tracking/run-dataset/index.js [hunk 1]
```diff
@@ -94,7 +95,7 @@ function buildDatasetDataMarkup(
             })}
             key={value.runId + index}
           >
-            {value.value}
+            {typeof value === 'object' ? JSON.stringify(value) : value.value}
```
Comment (studioswong) [inline src/components/experiment-tracking/run-dataset/index.js:98]: This is a temporary fix to resolve values that returns an object ( which is different from the design that the FE built with assumption on)

Comment (tynandebold) [reply src/components/experiment-tracking/run-dataset/index.js:98]: Got it.

Can you please push your changes you added when we were sitting together at your desk?

File: src/components/experiment-tracking/run-metadata/index.js [hunk 2]
```diff
@@ -15,20 +15,20 @@ const RunMetadata = ({ isSingleRun, runs = [] }) => {
     setToggleNotes({ ...toggleNotes, [index]: !toggleNotes[index] });
   };
 
-  return (
+  return runs.length === 0 ? (
+    <div>Loading...</div>
```
Comment (tynandebold) [inline src/components/experiment-tracking/run-metadata/index.js:19]: Maybe this `div` should have the same class names as the one below so it receives the same spacing?
```suggestion
    <div className={classnames('details-metadata', {
        'details-metadata--single': isSingleRun,
      })}>Loading...</div>
```

**Review by tynandebold — APPROVED** [review]
Looks good :)

Just one small comment about adding class names to a `div`.

**Pull request merged**